sympy/geometry/tests/test_ellipse.py

```diff
@@ -340,3 +342,38 @@ def test_reflect():
     assert e.area == -e.reflect(Line((1, 0), slope=0)).area
     assert e.area == -e.reflect(Line((1, 0), slope=oo)).area
     raises(NotImplementedError, lambda: e.reflect(Line((1, 0), slope=m)))
+
+
+def test_is_tangent():
+    e1 = Ellipse(Point(0,0), 3, 5)
+    c1 = Circle(Point(2, -2), 7)
+    assert e1.is_tangent(Point(0, 0)) is False
```

Comment:
Yes, I know about `contains`, but I don't know what is better, throw to User exception or just get answer, despite of doubts from geometrical point of view. 

Unfortunately, classes which also could have method `is_tangent` (I mean `Polygon`, `Parabola`, `Curve`) they don't have.